app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml

```diff
@@ -8,11 +8,10 @@
             %i.fas.fa-key{title:"Private Ontology"}
           - ontology_status = status_string(ontology)
           = render Display::InfoTooltipComponent.new(text: ontology_status, icon: submission_status_icons(ontology_status)) if session[:user]&.admin?
-      .browse-desc-container{data:{controller:"text-truncate", 'text-truncate-more-text-value': '+ Show more ...' , 'text-truncate-less-text-value': '- Show less ...'}}
+      .browse-desc-container{data:{controller:"text-truncate", 'text-truncate-more-text-value': '...+' , 'text-truncate-less-text-value': '...-'}}
```

Comment:
to confirm with @jonquet , it is less visible like this (users will not see it)

Reply:
Yes, that's what I see too